libraries/adaptive-expressions/src/triggerTrees/trigger.ts

```diff
@@ -18,13 +18,17 @@ import { TriggerTree } from './triggerTree';
 
 /**
  * Rewrite the expression by pushing not down to the leaves.
+ *
+ * @param expression Expression to rewrite.
+ * @param inNot .
```

Comment:
Is this description going to be just a dot?

Reply:
I couldn't figure out what this parameter is used for, and in .net it doesn't exist.